src/14-i2c/read-several-registers.md

```diff
@@ -53,7 +53,7 @@ Putting it all together inside a loop alongside a delay to reduce the data throu
 use aux14::{entry, iprint, iprintln, prelude::*};
 
 // Slave address
-const MAGNETOMETER: u8 = 0b001_1110;
+const MAGNETOMETER: u16 = 0b0011_1100;
```

Comment:
The address should not change. This adds a further `0` on the back instead of on the front. I would keep it was it was.

Reply:
I changed this because there are some changes between [`sadd1`](https://docs.rs/stm32f30x/0.7.0/stm32f30x/i2c1/cr2/struct.W.html#method.sadd1) and [`sadd`](https://docs.rs/stm32f3/0.13.0/stm32f3/stm32f303/i2c1/cr2/type.W.html#method.sadd). If I keep it as it was, there is no output in [14.4](https://docs.rust-embedded.org/discovery/14-i2c/the-solution.html), so I changed it to `0b0011_1100`, the output is exact `0x0A - 0b01001000`, the same as the example.

Reply:
Ah, I see. Does that mean that you tested this on real hardware?

Reply:
Year, I have a STM32F3 on hand. And I have tested every file that I have modified.

Reply:
Great then!